bftengine/src/bcstatetransfer/RVBManager.hpp

```diff
@@ -159,6 +160,7 @@ class RVBManager {
   std::vector<RVBGroupId> stored_rvb_digests_group_ids_;
 
   // RVB data update during checkpointing / pruning
+  concord::util::ThreadPool digests_worker_pool_{"digests-thread-pool", 8};
```

Comment:
I see that you are still creating a new worker pool. Did you consider re-using an existing one?